app/views/albums/edit.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="albums__topNav">
+  <%= button_to 'Home', root_path, method: 'get', class: "albums__homeButton" %>
+</div>
+
+<% if @album.errors.messages.present? %>
+  <%= render partial: "errors", locals: { record: @album } %>
```

Comment:
I made numerous attempts at not using locals: {}, but the only way I could get it to pass in was to have it there.